lib/mixins/element-mixin.html

```diff
@@ -14,6 +14,7 @@
 <link rel="import" href="../utils/case-map.html">
 <link rel="import" href="../utils/style-gather.html">
 <link rel="import" href="../utils/resolve-url.html">
+<link rel="import" href="../utils/html-fn.html">
```

Comment:
I think we decided to bundle the `html` tag function with the entry points, so that in modules a user can do e.g. `import {Element, html} from 'polymer-element.js';` as a convenience.

So, let's remove the import here, and add it to the two main end-user entry points (`polymer.html` and `polymer-element.html`).  In those, add `Polymer.html = Polymer.html;` so that modulizer adds exports for the `html` function from those.